packages/web-react/src/components/Button/useButtonStyleProps.ts

```diff
@@ -12,16 +12,16 @@ const getButtonColorClassname = <C = void>(className: string, color: ButtonColor
 const getButtonSizeClassname = <S = void>(className: string, size: ButtonSize<S>): string =>
   compose(applySize<ButtonSize<S>>(size))(className);
 
-export interface ButtonStyles<T> {
+export interface UseButtonStylesReturn {
```

Comment:
Please, do not rename this as you are not consistent. Even with your own style. Now there are `StyleReturn`, `StylePropsReturn`, and now `StylesReturn` suffixes. Stick with the `Styles` suffix as it is most used now. Thanks.